Readme.md

```diff
@@ -1,84 +1,195 @@
+</div>
+
+
+<br/>
+<br/>
+
+
+<div align="center" id="top">
+<table>
+  <tr>
+    <td valign="top" style="width:30%">
+      <details open="open">
+  <summary>Table of Contents</summary>
+
+  - [About](#-about)
+    - [Built With](#-built-with)
+  - [Getting Started](#-getting-started)
+    - [Install & Run](#-install)
+  - [Contributions](#%EF%B8%8F-contributions)
+  - [Acknowledgements](#-acknowledgements)
+
+  </details>
+    </td>
+    <td valign="top" style="width:70%"><img src="docs/images/coolkidsstayinghome.png" alt="Two girls sitting on a sofa"/></td>
+  </tr>
+</table>
+</div>
+
+
+<br/>
+<br/>
+
+<div align='center'>
+  <img src="docs/images/divider.png">
+</div>
+
+# 📢 About
 
 Together is a new app designed for 100devs. Together is a group calendar for public events, so you can see what's going on in your community. We're making it easier to find out about the events that are happening in Discord and let you know how much fun they'll be!
 
-**Link to project:** https://together.cyclic.app/
+<br/>
 
 ![alt tag](https://i.ibb.co/vVH3qjx/Screen-Shot-2022-11-05-at-12-13-17-PM.png)
 
-## How It's Made:
+<div align='center'>
+  <img src="docs/images/divider.png">
+</div>
+
+## ✅ Built With
+
+<div style="width:60%;margin:0 auto;" align="center">
+  <table>
+    <tr>
+      <td valign="center">
+      <img width="100%" title="React" src="https://img.shields.io/badge/React-0e062a?style=for-the-badge&logo=react&logoColor=61DAFB" alt="React"/>
+      <img width="100%" title="Express" src="https://img.shields.io/badge/Express.js-404D59?style=for-the-badge" alt="Express.js"/>
+      </td>
+      <td valign="center">
+       <img width="100%" title="Tailwind CSS" src="https://img.shields.io/badge/Tailwind_CSS-38B2AC?style=for-the-badge&logo=tailwind-css&logoColor=white" alt="Tailwind CSS"/>
+       <img width="100%" title="JavaScript" src="https://img.shields.io/badge/JavaScript-F7DF1E?style=for-the-badge&logo=JavaScript&logoColor=white" alt="JavaScript"/>
+      </td>
+      <td valign="center">
+       <img width="100%" title="Node.js" src="https://img.shields.io/badge/Node.js-90c53f?style=for-the-badge&logo=node.js&logoColor=white" alt="Node.js"/>
+       <img width="100%" title="MongoDB" src="https://img.shields.io/badge/MongoDB-4EA94B?style=for-the-badge&logo=mongodb&logoColor=white" alt="MongoDB"/>
+      </td>
+    </tr>
+  </table>
+</div>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+<br>
+<div align='center'>
+  <img src="docs/images/divider.png">
+</div>
+
+# 🚀 Getting Started
+## 💻 Install
 
-**Tech used:** Tailwind, JavaScript, MERN
-
-Work in Progress
-
-## Optimizations
-
-_(optional)_
-
-Work in Progress
+- ensure `.env` file is located in `server/config` file.
+- `npm install` in the root folder. installs the required dependencies.
+- `npm run build` compiles the build folder. Lint must have no errors for build file command to complete. 
```

Comment:
This line can be removed, as this is legacy instructions.

Reply:
Hey Caleb, I wasn't sure which line were you referring to here so I based myself on the '[Development Machine Setup](https://github.com/Caleb-Cohen/Together/wiki/02---Onboarding#development-machine-setup)' section from the wiki instead for the instructions.